_data/whats-new.yml

```diff
@@ -4,10 +4,202 @@ description: |
+    queries in the _GraphQL Developer Guide_.
+  versions: 2.3.3
+  type: Major update
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/5048
+- description: Added a description of the `@magentoConfigFixture` annotation to the
+    [GraphQL functional testing](https://devdocs.magento.com/guides/v2.3/graphql/functional-testing.html)
+    topic.
+  versions: 2.3.3
+  type: Major update
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/5086
+- description: Related items, up-sell, and cross-sell products are now supported in
+    the [products query](https://devdocs.magento.com/guides/v2.3/graphql/reference/products.html).
+  versions: 2.3.3
+  type: Major update
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4986
+- description: Added catalog attributes to the [storeConfig query](https://devdocs.magento.com/guides/v2.3/graphql/reference/store-config.html)
+    in the _GraphQL Developer Guide_.
+  versions: 2.3.x
+  type: Major update
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4920
+- description: Added the [PayPal Payments Advanced payment method](https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/payments-advanced.html)
+    topic to the _GraphQL Developer Guide_.
+  versions: 2.3.3
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/5035
+- description: Added the [checkoutAgreements query](https://devdocs.magento.com/guides/v2.3/graphql/queries/checkout-agreements.html)
+    topic to the _GraphQL Developer Guide_.
+  versions: 2.3.3
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4943
+- description: Updated the [customer query](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html)
+    topic in the _GraphQL Developer Guide_ to include store credit information.
+  versions: 2.3.3
+  type: Major update
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4993
+- description: Added the [Message queues and stores](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/message-queues/message-queues-stores.html)
+    topic to the _PHP Developer Guide_.
+  versions: 2.3.2
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4882
+- description: Added documentation for the [applyStoreCreditToCart](https://devdocs.magento.com/guides/v2.3/graphql/mutations/apply-store-credit.html)
+    and [removeStoreCreditFromCart](https://devdocs.magento.com/guides/v2.3/graphql/mutations/remove-store-credit.html) to the _GraphQL Developer Guide_.
+  versions: 2.3.3
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4960
+- description: Added documentation for the [setPaymentMethodAndPlaceOrder](https://devdocs.magento.com/guides/v2.3/graphql/reference/set-payment-place-order.html)
```

Comment:
The link redirects to home page

Reply:
Removed this item per instructions from @keharper